models/exceptions.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+"""
+module to handle all custom exceptions.
+"""
+
+class apiCallNonResposive(Exception):
+    """Exception raised when an API call returns non 200
+    """
+
+    def __init__(self,
+                 message="API call did not return a 200 reponse"):
+        """init for exception"""
+        
+        self.message = message
+        super().__init__(self.message)
+
+    def __str__(self):
+        """str definition"""
+        return ('{}'.format())
```

Comment:
Use f-string if you are using Python > 3.4 (I'm sorry again that the Checker is not enforcing their use, we are working on it).